server/bleep/src/webserver/intelligence.rs

```diff
@@ -335,14 +307,52 @@ pub(super) async fn token_value(
     Ok(json(TokenValueResponse { range, content }))
 }
 
+pub async fn get_token_info(
+    params: TokenInfoRequest,
+    repo_ref: &RepoRef,
+    indexes: Arc<Indexes>,
+    source_doc: &ContentDocument,
+    all_docs: &Vec<ContentDocument>,
+) -> anyhow::Result<Vec<FileSymbols>> {
+    let source_document_idx = all_docs
+        .iter()
+        .position(|doc| doc.relative_path == source_doc.relative_path)
+        .unwrap(); // FIXME: handle this
```

Comment:
can we preserve the orignal error handling code here? `.ok_or(anyhow::anyhow!("invalid language"))`?

Reply:
done